.mbedignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
This should probably not be in the root, and was meant for `bare_metal_tests`

Reply:
Content of the .mbedignore file has been modified to "bare_metal_tests/*".
This should allow to run bear metal tests and also regular tests.